2022/Q3/artifacts/PoS-pseudocode/PoS-model.md

```diff
@@ -294,9 +294,15 @@ func withdraw(validator_address, delegator_address)
   var delunbonds = {<start,end,amount> | amount = unbonds[delegator_address][validator_address].deltas[(start, end)] > 0 && end <= cur_epoch }
   //substract any pending slash before withdrawing
   forall (<start,end,amount> in selfunbonds) do
-    var amount_after_slashing = amount
```

Comment:
Was this mistakenly removed? L305 below updates `amount_after_slashing`, except now this variable isn't declared explicitly anywhere

Reply:
My fault, the intention is that line 305 declares it (just missing the `var` keyword). Fixed. Thanks ;)